go/pkg/rexec/rexec.go

```diff
@@ -384,6 +385,30 @@ func (ec *Context) DownloadOutputs(outputDir string) {
 	}
 }
 
+// GetOutputFileDigests returns a map of output file paths to digests.
+// This function is supposed to be run after a successful cache-hit / remote-execution
+// has been run with the given execution context. If called before the completion of
+// remote-execution, the function returns a nil result.
+func (ec *Context) GetOutputFileDigests(useAbsPath bool) (map[string]digest.Digest, error) {
+	if ec.resPb == nil {
+		return nil, nil
+	}
+
+	ft, err := ec.client.GrpcClient.FlattenActionOutputs(ec.ctx, ec.resPb)
+	if err != nil {
+		return nil, err
+	}
+	res := map[string]digest.Digest{}
+	for p, t := range ft {
```

Comment:
use more comprehensive names "filepath", "filetree", "tree" (??). I'm not as familiar with the code so it's ambiguous

Reply:
Done.